<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPatient</class>
 <widget class="QWidget" name="SearchPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>931</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>性别：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_sex">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>女</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电话：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phone">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>身份证号码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IDCard">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>就诊时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>就诊医师：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_doctor">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_outDate">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>169</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>主诉：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_mainAppeal">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>搜索结果：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_newPatient">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>120</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>新建患者</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>911</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
